src/sgame/sg_bot_ai.cpp

```diff
@@ -1289,7 +1289,13 @@ static bool build( gentity_t *self, buildable_t toBuild )
 		G_ForceWeaponChange( self, WP_HBUILD );
 	}
 
-	self->client->ps.stats[ STAT_BUILDABLE ] = toBuild;
+	if ( SB_BUILDABLE_TO_IBE( self->client->ps.stats[ STAT_BUILDABLE ] ) != IBE_NONE )
+	{
+		return false;
+	}
+
+	self->client->ps.stats[ STAT_BUILDABLE ] &= ~SB_BUILDABLE_MASK;
```

Comment:
The buildable UI error state is only updated every 100 ms. Also the use of `|=` doesn't make sense here.

Reply:
I try to set only the bits indicated in `SB_BUILDABLE_MASK` to `toBuild`. Is this wrong?

Reply:
Never mind I misread it.